### PR TITLE
Hackaton/chart live data

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Adds a chart data from the GraphQL.
If there is no chart data it will show default chart line
Also adds a chart loading animation.

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @nenadV91 , looks like there is an issue with loading data from bff 
image

Also, not sure if it is OK to display a curve (default) line if there is no data. I'd replace it with an explanation text. Something like "Error loading chart data. Please try again later' or so. 
WDYT?

**Comment by elena-zh** [comment]
Also, another my point that an empty area takes too much room on the page

image

File: .env [hunk 1]
```diff
@@ -1,2 +1,3 @@
 DUNE_API_KEY=tuBkOcda6ymMvmnpIX9Qer9Xb5hJ2rf8
-NEXT_PUBLIC_GOOGLE_ANALYTICS=
\ No newline at end of file
+NEXT_PUBLIC_GOOGLE_ANALYTICS=
+NEXT_PUBLIC_AWS_API_ENDPOINT="https://bff.barn.cow.fi/proxy"
```
Comment (anxolin) [inline .env:3]: shouldn't this env have some `BFF` in it? (also, no need to have AWS

**Review by anxolin — COMMENTED** [review]
👍

**Pull request merged**